bundles/org.openhab.binding.mynice/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,165 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="mynice"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<bridge-type id="it4wifi">
+		<label>IT4Wifi</label>
+		<description>This thing connects to your IT4Wifi module</description>
+
+		<representation-property>macAddress</representation-property>
+
+		<config-description>
+			<parameter name="username" type="text">
+				<label>Username</label>
+				<description>User defined on the IT4Wifi for openHAB</description>
+			</parameter>
+			<parameter name="hostname" type="text" required="true">
+				<context>network-address</context>
+				<label>Hostname</label>
+				<description>Hostname or IP address of the device</description>
+			</parameter>
+			<parameter name="macAddress" type="text" pattern="([0-9A-Fa-f]{2}:){5}[0-9A-Fa-f]{2}(\s*#.*)*"
+				required="true">
+				<label>MAC Address</label>
+				<description>The MAC address of the IT4Wifi</description>
+			</parameter>
+			<parameter name="password" type="text">
+				<context>password</context>
+				<label>Pairing Key</label>
+				<description>Pairing Key needed to access the device, provided by the bridge itself</description>
+			</parameter>
+		</config-description>
+	</bridge-type>
+
+	<thing-type id="swing">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="it4wifi"/>
+		</supported-bridge-type-refs>
+
+		<label>Swing Gate</label>
+		<description>A double wing gate</description>
```

Comment:
wing or swing?

Reply:
Changed to dual swing gate that seems more convenient in english